thrimshim/thrimshim/main.py

```diff
@@ -236,28 +236,30 @@ def get_row(ident):
 	if is_archive:
 		response["allow_holes"] = True
 
-	# pick default thumbnail template based on start time.
-	# pick default frame time as the middle of the video.
-	# ignore both if video has no start time yet.
-	# or for archive events, no thumbnail.
-	DEFAULT_TEMPLATES = [
-		"zeta-right",
-		"dawnguard-right",
-		"alphaflight-right",
-		"nightwatch-right",
-	]
 	if is_archive:
 		response["thumbnail_mode"] = "NONE"
 	elif response['event_start'] is not None:
 		start = response['event_start']
-		if response['thumbnail_template'] is None:
-			# RDPs default to the RDP template. Others use the current shift.
-			if response['category'] == "RDP":
-				response['thumbnail_template'] = "rdp"
-			else:
-				pst_hour = (start.hour - 8) % 24
-				shift = int(pst_hour / 6)
-				response['thumbnail_template'] = DEFAULT_TEMPLATES[shift]
+
+		# use tags to determine default thumbnail template
+		if response['thumbnail_template'] is None
+			conn = app.db_manager.get_conn()
+			query = """
+				SELECT name, default_template
+				FROM playlists
+					"""
+			results = database.query(conn, query)
+			default_templates = {}
+			for row in results:
+				row = row._asdict:
```

Comment:
Generally it's easier to work with the row as a namedtuple instead of a dict, unless you need to pass it to something that expects a dict, or iterate over keys etc.
So for example, without this line you could write `row.name` instead of `row['name']`.